modules/ROOT/pages/appendix/gql-conformance/additional-cypher.adoc

```diff
@@ -507,17 +507,17 @@ GQL supports `GRAPH TYPES` as a way of constraining a graph schema, but does not
 | Cypher feature
 | Description
 
-| xref:constraints/managing-constraints.adoc#create-uniqueness-constraints[Uniqueness constraints]
-| Ensure that property values are unique for all nodes with a specific label or all relationships with a specific type. 
+| xref:constraints/managing-constraints.adoc#create-property-uniqueness-constraints[Property uniqueness constraints]
+| Ensures that the combined property values are unique for all nodes with a specific label or all relationships with a specific type.
 
-| xref:constraints/managing-constraints.adoc#create-existence-constraints[Existence constraints]
-| Ensure that a property exists either for all nodes with a specific label or for all relationships with a specific type.
+| xref:constraints/managing-constraints.adoc#create-property-existence-constraints[Property existence constraints]
+| Ensures that a property exists either for all nodes with a specific label or for all relationships with a specific type.
 
-| xref:constraints/managing-constraints.adoc#create-type-constraints[Type constraints]
-| Ensure that a property has the required property type for all nodes with a specific label or for all relationships with a specific type.
+| xref:constraints/managing-constraints.adoc#create-property-type-constraints[Property type constraints]
+| Ensures that a property has the required property type for all nodes with a specific label or for all relationships with a specific type.
 
-| | xref:constraints/managing-constraints.adoc#create-key-constraints[Key constraints]
-| ensure that, for a given node label or relationship type and set of properties, all properties are present on the respective nodes or relationships, and the combination of property values is unique.
+| xref:constraints/managing-constraints.adoc#create-key-constraints[Key constraints]
+| Ensures that property values are unique and exists for all nodes with a specific label or all relationships with a specific type. 
```

Comment:
```suggestion
| Ensures that the combined property values are unique and all exists for all nodes with a specific label or all relationships with a specific type.
```
maybe? to better match the updated property uniqueness one? 🤔 

or maybe in the other order if that is better grammar
```suggestion
| Ensures that all properties exist and that the combined property values are unique for all nodes with a specific label or all relationships with a specific type.
```